.babelrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+    env: {
+        pure: {},
+        browser: {
+            presets: [
+                ["@babel/preset-env", {
+                    useBuiltIns: "usage",
+                    modules: false,
+                    corejs: {
+                        version: 3
+                    },
+                    targets: [
+                        "last 2 Chrome versions",
+                        "last 2 firefox versions",
+                        "last 2 Edge versions",
+                        "ie 10-11"
+                    ],
+                    // debug: true,
+                    loose: true, // needed for IE 10
+                }]
+            ]
+        },
+        development: {
```

Comment:
Why did you remove the `@babel/plugin-transform-runtime` plugin. I used it to make the bundle smaller but maybe you had a good reason to remove it?

Reply:
I didn't see any size reduction when I was inspecting the generated code with and without the plugin. The bundle is actually ~5K bigger with the transform.